### DISCUSSION TITLE
2024 Brazilian Portuguese PT-BR Translation

### DISCUSSION THREAD

**Discussion by 1stsongmusic** [aclap-dev/video-downloadhelper#891] [Language & Translation]
Hi 

I'm sending the non-translated strings for PT-BR

Hope You All Enjoy It!

[2024 pt_BR Video DownloadHelper.txt](https://github.com/user-attachments/files/15882452/2024.pt_BR.Video.DownloadHelper.txt)

**Comment by paulrouget** [comment]
Sorry we already got a translation for that language 🙏

I recommend that you try the upcoming version (v9.1 which should be released next week) see if the translation is correct.

If you want to keep track of the current translation status of your language, I recommend following this github repository: https://github.com/aclap-dev/vdhlocales
